conformance-suites/2.0.0/js/tests/tex-image-and-sub-image-with-image-bitmap-utils.js

```diff
@@ -333,76 +333,83 @@ function runOneIterationImageBitmapTestSubSource(useTexSubImage, bindingTarget,
     wtu.glErrorShouldBe(gl, gl.NO_ERROR, "should be no errors");
 }
 
-function runTestOnBindingTargetImageBitmap(bindingTarget, program, bitmaps, optionsVal,
+function runTestOnBindingTargetImageBitmap(bindingTarget, program, cases, optionsVal,
     internalFormat, pixelFormat, pixelType, gl, tiu, wtu)
 {
-    var cases = [
-        { sub: false, bitmap: bitmaps.noFlipYPremul, flipY: false, premultiply: true },
-        { sub: true, bitmap: bitmaps.noFlipYPremul, flipY: false, premultiply: true },
-        { sub: false, bitmap: bitmaps.noFlipYUnpremul, flipY: false, premultiply: false },
-        { sub: true, bitmap: bitmaps.noFlipYUnpremul, flipY: false, premultiply: false },
-        { sub: false, bitmap: bitmaps.flipYPremul, flipY: true, premultiply: true },
-        { sub: true, bitmap: bitmaps.flipYPremul, flipY: true, premultiply: true },
-        { sub: false, bitmap: bitmaps.flipYUnpremul, flipY: true, premultiply: false },
-        { sub: true, bitmap: bitmaps.flipYUnpremul, flipY: true, premultiply: false },
-    ];
-
-    for (var i in cases) {
-        runOneIterationImageBitmapTest(cases[i].sub, bindingTarget, program, cases[i].bitmap,
-            cases[i].flipY, cases[i].premultiply, optionsVal, internalFormat, pixelFormat, pixelType, gl, tiu, wtu);
-    }
+    cases.forEach(x => {
+        runOneIterationImageBitmapTest(x.sub, bindingTarget, program, x.bitmap,
+            x.bitmap.flipY, x.bitmap.premultiply, optionsVal, internalFormat, pixelFormat, pixelType, gl, tiu, wtu);
+    });
 
     if (wtu.getDefault3DContextVersion() > 1 &&
-        (bindingTarget == gl.TEXTURE_2D || bindingTarget == gl.TEXTURE_3D)) {
-        // SKip testing source sub region on TEXTURE_CUBE_MAP and TEXTURE_2D_ARRAY to save running time.
-        for (var i in cases) {
-            runOneIterationImageBitmapTestSubSource(cases[i].sub, bindingTarget, program, cases[i].bitmap,
-                cases[i].flipY, cases[i].premultiply, optionsVal, internalFormat, pixelFormat, pixelType, gl, tiu, wtu);
-        }
+        (bindingTarget == gl.TEXTURE_CUBE_MAP || bindingTarget == gl.TEXTURE_2D_ARRAY))
+    {
+        // Skip testing source sub region on TEXTURE_CUBE_MAP and TEXTURE_2D_ARRAY on WebGL2 to save
+        // running time.
+        return;
     }
+
```

Comment:
This code snippet above is a little different from the original one. 

In the original code, it will never execute runOneIterationImageBitmapTestSubSource() under WebGL 1 context. 

However, after this change is applied, it will execute runOneIterationImageBitmapTestSubSource() in WebGL 1 context when bindTarget is gl.TEXTURE_2D or gl.TEXTURE_3D. This make Chromium failed to find texImage2D API in WebGLRenderingContext and then context lost finally. (I am not familiar with sub resource related code, I suppose that sub resource is not supported in webgl 1?) . 

See the small change at https://github.com/KhronosGroup/WebGL/pull/2251/.